KesaApplication/ViewController.swift

```diff
@@ -8,18 +8,180 @@
+    @IBOutlet weak var password: UITextField!
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+        
+        firstName.delegate = self // set delegate to 4 text fields
+        lastName.delegate = self
+        email.delegate = self
+        password.delegate = self
+        
     }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+    
+    @IBAction func onLoginButtonClickListener(sender: UIButton) {
+        
+        /* Once login button is pressed, text fields are no longer important */
+        self.firstName.resignFirstResponder() // Give up first responder status
+        self.lastName.resignFirstResponder()
+        self.email.resignFirstResponder()
+        self.password.resignFirstResponder()
+    }
+    
+    
+    /* When the screen is touched */
+    override func touchesBegan(touches: Set<UITouch>,
+                               withEvent event: UIEvent?) {
+        self.view.endEditing(true) /* For instances where the keyboard is 
+                                      already brought up, Hide the keyboard */
+        firstNameLabel.textColor = UIColor.blackColor()
+        lastNameLabel.textColor = UIColor.blackColor()
+        emailLabel.textColor = UIColor.blackColor()
+        passwordLabel.textColor = UIColor.blackColor()
     }
+    
+    
+    
+    // Highlighting each labels when their text fields are chosen
+    func textFieldDidBeginEditing(textField: UITextField) {
+        
+        if (textField == firstName) {
+            firstNameLabel.textColor = UIColor.orangeColor()
+            lastNameLabel.textColor = UIColor.blackColor()
+            emailLabel.textColor = UIColor.blackColor()
+            passwordLabel.textColor = UIColor.blackColor()
+        }
 
+        else if (textField == lastName) {
+            firstNameLabel.textColor = UIColor.blackColor()
+            lastNameLabel.textColor = UIColor.orangeColor()
+            emailLabel.textColor = UIColor.blackColor()
+            passwordLabel.textColor = UIColor.blackColor()
+        }
+        
+        else if (textField == email) {
+            firstNameLabel.textColor = UIColor.blackColor()
+            lastNameLabel.textColor = UIColor.blackColor()
+            emailLabel.textColor = UIColor.orangeColor()
+            passwordLabel.textColor = UIColor.blackColor()
+        }
 
+        else {
+            firstNameLabel.textColor = UIColor.blackColor()
+            lastNameLabel.textColor = UIColor.blackColor()
+            emailLabel.textColor = UIColor.blackColor()
+            passwordLabel.textColor = UIColor.orangeColor()
+        }
+        
+    }
+    
+    
+    @IBAction func onCreateAccountButtonClickListener(sender: UIButton) {
+        
+        let utility = Utility() // An instance of the Utility Class
```

Comment:
After changing to static methods, this instantiation is not required. Also the comment is unnecessary here.